packages/circuit-ui/components/ImageInput/ImageInput.tsx

```diff
@@ -478,7 +478,6 @@ export const ImageInput = ({
       </InputWrapper>
       <FieldValidationHint
         id={validationHintId}
-        aria-live="polite"
```

Comment:
Ah, I forgot it had been added explicitly, thought it was my doing in the initial implementation of the `ImageInput`. My reasoning is in the commit desc (f7385c5):

> A live region only makes sense when the validationHint is an error, not a description. Further research and design are required to improve the field-level validation pattern. Removing the live region and will reintroduce in the future when consistent with other form components.

Basically the hint shouldn't _always_ be in a live region, and I am planning on revisiting this after I talk with design about splitting errors from descriptions (they're not the same).

Alternatively if we'd like to keep this in the interim, I could suggest a simple in-between solution:

```diff
- aria-live="polite"
+ aria-live={invalid ? "polite" : undefined}
```

The only issue here is that for optimal compatibility, live regions should be rendered _before_ their content, not at the same time. I've never come across this issue myself, but it's a known edge case.

Reply:
Quick testing in VO on Chrome, the hint doesn't get announced on page load (as expected). It still might if the input is conditionally rendered. But as discussed offline, I'll leave the live region in place for now, this will be revisited holistically in the future as mentioned above.

Reply:
This is fantastic 👏

Reply:
Tests will look like this (done for the Checkbox component) https://github.com/sumup-oss/circuit-ui/pull/1795/commits/379d3aabb7f265d7b148daa36d34a476c9e5fffe?w=1

Reply:
I'm debating whether it would make sense to add these tests to the `FieldValidationHint` only, instead of all components? Would be less work, but wouldn't cover the outer API layer, only the internal unit 🤔

Reply:
I'd rather cover components since for instance it's important to keep the association between the description and input using `aria-describedby` and an `id`. Don't mind the extra work, i'm also improving some older specs in the repo on the way 🙂

Reply:
Good point. Maybe there's a way to abstract these tests so you don't have to repeat the same boilerplate over and over?